src/index.js

```diff
@@ -90,10 +98,13 @@ DecafAppModule.config((appNameProvider, appAuthProvider, potionProvider, decafAP
         };
         if (Session.isAuthenticated()) {
             setRavenUser();
+        } else {
+            $window.localStorage.setItem('sessionJWT', process.env.GUEST_TOKEN);
```

Comment:
This one also works, but I meant to use it as it listed here:

https://github.com/grevory/angular-local-storage#api-documentation

I'm not insisting on this though